.github/workflows/custom_objectdetection_ci_pipeline.yml

```diff
@@ -0,0 +1,34 @@
+name: Custom Object Detection CI Workflow
+
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - mlops/common/**
+      - mlops/**
```

Comment:
i might be wrong, but doesn't this line 9 effectively recursively cover all the content in the `mlops` directory? making the other lines not necessary?

Reply:
yes.. I updated all of the pipelines. good catch!